domains/auth/Auth.API/API/Controllers/TermsController.cs

```diff
@@ -0,0 +1,32 @@
+using API.Models;
+using API.Services;
+using API.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[Authorize]
+[ApiController]
+public class TermsController : ControllerBase
+{
+    [HttpPut()]
+    [Route("terms/accept")]
+    public async Task<ActionResult<User>> AcceptTermsAsync([FromBody] int acceptedTermsVersion, [FromServices] IUserDescriptMapper descriptMapper, [FromServices] IUserService userService)
```

Comment:
We should map a version property from a Json object sent in the body instead of `[FromBody] int acceptedTermsVersion`.